In [1]:
import os

In [2]:
import pandas as pd

In [3]:
data = pd.DataFrame(columns = ['userId', 'movieId', 'rating', 'timestamp'])

In [4]:
files = os.listdir('data')

In [5]:
files

['ratings_9.txt',
 'ratings_4.txt',
 'ratings_5.txt',
 'ratings_1.txt',
 'ratings_10.txt',
 'ratings_3.txt',
 'ratings_6.txt',
 'ratings_7.txt',
 'ratings_8.txt',
 'ratings_2.txt']

#### Напишите цикл, который собирает содержимое файлов папки data в единый датафрейм data.

#### Сколько строк в датафрейме data?

#### Solution:

In [6]:
for filename in files:
    temp = pd.read_csv(os.path.join('data', filename), names = ['userId', 'movieId', 'rating', 'timestamp'])
    data = pd.concat([data, temp])

In [7]:
len(data)

100004

#### Conclusion: the code above is one of the right decisions and 100004 is the right answer!!